msticpy/context/tiproviders/ti_provider_base.py

```diff
@@ -32,13 +32,13 @@
 class TIProvider(Provider):
     """Abstract base class for Threat Intel providers."""
 
-    _IOC_QUERIES: Dict[str, Any] = {}
+    _QUERIES: Dict[str, Any] = {}
 
     # pylint: disable=unused-argument
     def __init__(self, **kwargs):
         """Initialize the provider."""
         # pylint: disable=invalid-name
-        self._QUERIES = self._IOC_QUERIES
+        self._QUERIES = self._QUERIES
```

Comment:
I'm sure this was there before but do you see any reason for it?
I mean the class attrib _QUERIES will always be self._QUERIES right?